docker-compose.yml

```diff
@@ -29,6 +29,13 @@ services: # the different images that will be running as containers
     restart: "on-failure"
     depends_on: # cannot start if db service is not up and running
       - db
+  redis: # service name
+    image: "redis:5" # image name of the redis database. during build, this will be pulled from dockerhub and a container spun up from it
+    hostname: redis # channel layer will connect to this hostname in settings.py
```

Comment:
why is this needed? Similar to `db`, wouldn't this automatically default to the service name?